src/content/homepage.md

```diff
@@ -203,30 +203,84 @@ milestonesSection:
         text: First dApp PoC
         when: 'Jan 2022'
         isMajor: true
-      - order: 1
+      - order: 2
         text: Desktop wallet upgrade
         when: 'Apr 2022'
         isMajor: false
-      - order: 2
+      - order: 3
         text: Explorer optimization & addition of statistics
         when: 'May 2022'
         isMajor: false
-      - order: 3
+      - order: 4
         text: Full node 1.4.0 with all of the major features for the Leman network upgrade implemented
         when: 'Jun 2022'
         isMajor: false
-      - order: 4
+      - order: 5
         text: Beta version of contract SDK (alephium-web3)
         when: 'Jun 2022'
         isMajor: false
-      - order: 5
+      - order: 6
         text: New testnet launched
         when: 'Jun 2022'
         isMajor: false
-      - order: 6
+      - order: 7
         text: Leman Network Upgrade deployed on the testnet
         when: 'Jun 2022'
         isMajor: true
+      - order: 8
+        text: Improve documentation of the core infrastructure
+        when: 'H2 2022'
+      - order: 9
+        text: Improve the Alephium SDK with more features
+        when: 'H2 2022'
+      - order: 10
+        text: Improve the robustness and efficiency of the full node &  explorer backend
+        when: 'H2 2022'
```

Comment:
I would merge those into one, smth along those lines: `Enhance the core platform's robustness and efficiency while improving documentation.`